src/couchdb.ts

```diff
@@ -113,6 +113,31 @@ export class CouchDb {
         });
     }
 
+    /**
+     * Get the list of specified databases info.
+     * @param {{
+     *         keys: string[];
+     *     }} options
+     * @return {Promise}
+     */
+    findDatabaseInfo(options: {
+        keys: string[]
```

Comment:
Lets not call this `keys` .. people wont really know what that means but looking at their docs, it appears to be db names so we should call this property `dbNames`

Reply:
Doesn't the endpoint need this to be names keys?  

Request JSON Object:
keys (array) – Array of database names to be requested

Reply:
Yes but that doesnt mean our function has to match their call signature